pkg/role_binding/role_binding.go

```diff
@@ -0,0 +1,127 @@
+package rolebinding
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/openshift-online/ocm-support-cli/pkg/types"
+)
+
+type RoleBinding struct {
+	types.Meta
+	ID             string
+	HREF           string
+	AccountID      string
+	RoleID         string
+	OrganizationID string `json:",omitempty"`
+	SubscriptionID string `json:",omitempty"`
+	CreatedAt      time.Time
+	UpdatedAt      time.Time
+	Type           string
+}
+
+const (
+	SubscriptionRoleBinding = "Subscription"
+	OrganizationRoleBinding = "Organization"
+	ApplicationRoleBinding  = "Application"
+)
+
+func CreateRoleBinding(accountID string, roleID string, roleType string, resourceID *string) (*v1.RoleBinding, error) {
+	newRoleBinding := v1.NewRoleBinding().AccountID(accountID).RoleID(roleID).Type(roleType)
+	if roleType == SubscriptionRoleBinding {
+		newRoleBinding.SubscriptionID(*resourceID)
+	}
+	if roleType == OrganizationRoleBinding {
+		newRoleBinding.OrganizationID(*resourceID)
+	}
+	rb, err := newRoleBinding.Build()
+	if err != nil {
+		return nil, fmt.Errorf("can't create role binding : %v", err)
+	}
+	return rb, nil
+}
+
+func AddRoleBinding(accountID string, roleID string, roleType string, resourceID *string, conn *sdk.Connection) (*v1.RoleBinding, error) {
+	rb, err := CreateRoleBinding(accountID, roleID, roleType, resourceID)
+	if err != nil {
+		return nil, fmt.Errorf("%v", err)
+	}
```

Comment:
CreateRoleBinding also returns a fmt.Errorf. Consider returning the err directly.